cmd/swarm/swarm-smoke/upload_and_sync.go

```diff
@@ -50,11 +56,77 @@ func uploadAndSyncCmd(ctx *cli.Context, tuid string) error {
 		metrics.GetOrRegisterCounter(fmt.Sprintf("%s.timeout", commandName), nil).Inc(1)
 
 		// trigger debug functionality on randomBytes
+		err := triggerChunkDebug(randomBytes)
```

Comment:
`randomBytes` could be potentially very large. I am assuming that the solution of using `TeeReader` here would not work so well because the debug functionality is only sometimes being used. It might be possible to pass a pointer in order to avoid copying the value.

Reply:
I am passing a slice now